vnext/Microsoft.ReactNative/ReactInstanceSettings.idl

```diff
@@ -151,6 +151,11 @@ namespace Microsoft.ReactNative
     DOC_DEFAULT("false")
     Boolean EnableByteCodeCaching { get; set; };
 
+    DOC_STRING(
+      "Opt in to using the default unhandled exception handler that logs additional information into a text file for WER.")
```

Comment:
If it is default, then it should be turned on? Otherwise it isn't the default :)

Reply:
A naming issue; maybe "baseline" or "reference implementation / built-in" instead of default?

Reply:
I wonder if should reverse it and have the DisableDefaultCrashHandler. I guess the main questions is if this default allowes third party developers to get the crash reports. If yes, then it should be 'on' by default.

Reply:
I'm not sure about this, the change could look invasive. I recommend starting with opt-in and seeing if we get any feedback on it.